src/main/java/vazkii/botania/common/item/lens/ItemLens.java

```diff
@@ -319,6 +319,6 @@ public ItemStack setCompositeLens(ItemStack sourceLens, ItemStack compositeLens)
 
 	@Override
 	public boolean shouldPull(ItemStack stack) {
-		return stack.getItemDamage() != STORM;
+		return stack.getItemDamage() < 5000;
```

Comment:
I was originally going to use a beam of mana bursts with a lens of damage 5001, but that caused a few issues that I didn't want to deal with.

I figured that line of code might be useful if you might ever want to implement specialized lens again.